orchestrator/api/agent_plugins.py

```
@@ -0,0 +1,363 @@
+                ).all()
+            }
+
+            not_enabled = [
+                str(pid) for pid in body.plugin_ids if pid not in enabled_plugin_ids
+            ]
+
+            if not_enabled:
+                raise HTTPException(
+                    status_code=400,
+                    detail=f"Plugins not enabled for workspace: {', '.join(not_enabled)}",
+                )
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)